CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # CHANGELOG
 
+## v0.26.1
+
+*January 16, 2023*
+
+This release contains a bug fix for the `ChanOpenConfirm` handler and strongly recommended to upgrade.
+
```

Comment:
Can you also add a statement that this is a consensus-breaking change in the channel handshake?

```
This release contains a consensus-breaking change during the channel opening handshake; it was broken, and now is fixed.
```

Reply:
Actually we should have both statements. It should read:

```
*January 16, 2023*

This release contains a bug fix for the `ChanOpenConfirm` handler and it is strongly recommended to upgrade.

The change is consensus-breaking (during the channel opening handshake); it was broken, and now is fixed.
```

Reply:
My bad... done